### DISCUSSION TITLE
Watermark

### DISCUSSION THREAD

**Discussion by Teddy937** [spatie/laravel-pdf#81] [Ideas]
Is there a way we can add watermarks in pdfs. Something like company signatures as watermarks

**Comment by ArielMejiaDev** [comment]
@Teddy937  As it is Html you can add a watermark using CSS, eg:

```
body {
  background-image: url('../watermark.png');
  background-size: contain; /* Make background take entire page */
  background-position: center; /* Center Background */
}
```

**Reply by Teddy937** [reply]
Thanks it works fine,, Background repeat can be set to no-repeat if you don't want them all over

**Reply by ArielMejiaDev** [reply]
@aldinokemal could you provide the blade code, as far as I see you should add watermark css class on every div wrapper after page break directive too

**Reply by aldinokemal** [reply]
@ArielMejiaDev, maybe need to add in body instead of class, problem solved

**Reply by ArielMejiaDev** [reply]
@aldinokemal I think so

**Comment by mrleblanc101** [comment]
I hope Watermark support is added, it is natively supported by Gotenberg
